packages/syft/src/syft/service/worker/worker_image_service.py

```diff
@@ -192,18 +194,7 @@ def get_all(self, context: AuthedServiceContext) -> DictTuple[str, SyftWorkerIma
         One image one docker file for now
         """
         images = self.stash.get_all(credentials=context.credentials).unwrap()
-
-        res = {}
-        # if image is built, index it by full_name_with_tag
-        for im in images:
-            if im.is_built and im.image_identifier is not None:
-                res[im.image_identifier.full_name_with_tag] = im
```

Comment:
Q:  Are we deleting the method indexing images by tag, as it is not useful, do we need to confirm with product ?

Reply:
I thing is that indexing by tag is a buggy with different images with same tag. This effects images.get_all() functionality. Let's remove this and inform this to the product, we can re-implement it as a separate ticket.